services/order.py

```diff
@@ -0,0 +1,35 @@
+from typing import List, Dict, Optional
+from django.db import transaction
+from db.models import Order, Ticket, MovieSession
+from django.contrib.auth import get_user_model
+
+
+@transaction.atomic
+def create_order(
+    tickets: List[Dict[str, int]], username: str, date: Optional[str] = None
+) -> Order:
+    user = get_user_model().objects.get(username=username)
```

Comment:
The `get` method will raise a `DoesNotExist` exception if no user with the given username is found. Consider handling this exception to provide a more user-friendly error message or behavior.